src/Symfony/Component/BrowserKit/Response.php

```diff
@@ -123,13 +123,14 @@ public function getHeaders()
      */
     public function getHeader($header, $first = true)
     {
+        $normalizedHeader = str_replace('-', '_', strtolower($header));
         foreach ($this->headers as $key => $value) {
-            if (str_replace('-', '_', strtolower($key)) == str_replace('-', '_', strtolower($header))) {
+            if (str_replace('-', '_', strtolower($key)) === $normalizedHeader) {
                 if ($first) {
                     return is_array($value) ? (count($value) ? $value[0] : '') : $value;
                 }
 
-                return is_array($value) ? $value : array($value);
+                return (array) $value;
```

Comment:
According to the Twig version check, the fix was implemented in PHP 5.4.0 (Twig puts a workaround for it only on older 5.3 and lower): https://github.com/twigphp/Twig/blob/eecf85e903322e425ea921fd2588238f632965d2/lib/Twig/Node/Expression/Name.php#L46